packages/stacked/lib/src/base_view_models.dart

```diff
@@ -175,10 +176,15 @@ abstract class FutureViewModel<T> extends _SingleDataSourceViewModel<T> {
 
   /// Called after the data has been set
   void onData(T data) {}
+
+  initialise() {
```

Comment:
Instead of calling runFuture from the initialise function we can instead change the runFuture function to be called initialise.

Reply:
Renamed the function.